leaf-ui/rappid-extensions/ConfigInspector.js

```diff
@@ -99,7 +99,7 @@ var Config = Backbone.View.extend({
 
     /** Create and render dropdown inner view, set listeners */
```

Comment:
Still need to remove this file from the commit - I can show you how if you want!

Reply:
Yes please show me!

Reply:
I will show you rn